### PR TITLE
ref(feedback): update integration test and doc for #3403

### PR DESCRIPTION
**Pull request by aliu39**
Follow-up from https://github.com/getsentry/relay/pull/3403.

#skip-changelog  (doc and renaming)

### CODE REVIEW

**Comment by aliu39** [comment]
Is there a way to bypass the changelog test for a small refactor like this?

File: tests/.idea/workspace.xml
Comment (jjbayer) [inline tests/.idea/workspace.xml:1]: Please delete this file.

**Review by jjbayer — APPROVED** [review]
You can add `#skip-changelog` to the PR description.

File: tests/.idea/workspace.xml
Comment (iker-barriocanal) [inline tests/.idea/workspace.xml:1]: If you use IDEs or tools that create specific files, you can tell git to [ignore these files](https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer) in all repos in your computer. You can still manually add them.

**Review by iker-barriocanal — APPROVED** [review]
Thanks for following up!

File: tests/.idea/workspace.xml
Comment (aliu39) [reply tests/.idea/workspace.xml:1]: Great, thanks for the tip! I only knew .gitignore before this

**Pull request merged**